index.d.ts

```diff
@@ -0,0 +1,10 @@
+declare module 'ru-validation-codes' {
+  export declare function checkINN(value: string | number): boolean;
```

Comment:
Do we really need **declare** keyword for the export function signature?

Reply:
@kdmatrosov 
When a library is used in a typescript project, this is a necessarily.
I think in this case, it would make sense in the package "@types/ru-validation-codes"